src/types/index.d.ts

```diff
@@ -37,7 +37,7 @@ import { $pres } from './builder.js';
  * @property {Object.<string, string>} NS
  * @property {XHTML} XHTML
  */
-export const Strophe: {
+declare const Strophe: {
```

Comment:
Did you make this change by hand? These files are autogenerated (by running `make types`).

Reply:
Thanks for the review, I reverted this part of the changes and pushed the newly generated version made by `make dist/types` command.